ferrocene/doc/sphinx-shared-resources/exts/ferrocene_domain_cli/__init__.py

```diff
@@ -27,5 +27,6 @@ def setup(app):
         #
         # Version history:
         # - 0: initial implementation
-        "env_version": "0",
+        # - 1: add no_traceability_matrix flag
+        "env_version": "1",
```

Comment:
Is is actually necessary to bump the `env_version`, since CI builds the docs from scratch anyways?

Reply:
So, in general yes, whenever we change the environment we should increase the env version. You are right that in CI we build the docs from scratch, but we also build docs incrementally on our developer machines too. Without incrementing the environment version we would get confusing error messages locally.

*But* in this specific case we don't need to bump the env version. We only changed `env.temp_data`, which is not persisted to disk between incremental builds and is cleared as soon as reading the document finished.